gender_novels/corpus_gen.py

```diff
@@ -451,6 +501,10 @@ def get_author_gender(authors):
     'female'
     >>> get_author_gender(['Shakespeare, William', "Duan, Mingfei"])
```

Comment:
```suggestion

    Because gender_guesser cannot guess the gender of "Mingfei" this reports "both":
    
    >>> get_author_gender(['Shakespeare, William', "Duan, Mingfei"])
```